pkg/deploy/config_test.go

```diff
@@ -42,7 +42,7 @@ func TestConfigurationFieldParity(t *testing.T) {
 		// check each parameter exists as a field in Configuration
 		for name := range params.Parameters {
 			switch name {
-			case "deployNSGs", "rpImage", "rpServicePrincipalId", "vmssName":
+			case "deployNSGs", "rpImage", "fluentbitImage", "rpServicePrincipalId", "vmssName":
```

Comment:
I'm not sure if this is necessary.  The `fluentbitImage` is already in the `config.Configuration` struct so this should pass without omitting this parameter.  
```suggestion
			case "deployNSGs", "rpImage", "rpServicePrincipalId", "vmssName":
```

Reply:
Fixed, thanks.